docs/content/community/getting-help.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Getting Help | Dagster
+description: Have questions about how to use Dagster? Hit a bug? Have a feature request? This page includes tips on what to do.
+---
+
+# Guide to Getting Help
+
+Have questions about how to use Dagster? Hit a bug? Have a feature request? The Dagster Labs team and the Dagster community make a best-effort attempt to respond, on a few different platforms.
+
+For Dagster Cloud users, an Enterprise contract also comes with a guaranteed SLA.
+
+### Searching for answers
+
+Search engines like Google generally do a good job at surfacing relevant Dagster docs, Github Discussions and Github Issues, so we recommend starting there.
+
+Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster, because the corpus it was trained on doesn’t include Dagster’s latest APIs.
+
+### Bugs and feature requests go in Github Issues
+
+We use [Github Issues](https://github.com/dagster-io/dagster/issues) to track all bugs and feature requests from the community.
+
+If you have a request of the form, “I want Dagster to do X, but it seems like it doesn’t”, consider filing a Github Issue instead of posting a Slack message.  In most cases, our response on Slack will be to ask you to file a Github Issue, which makes it easier to track your request.
+
+If you find a Github issue that seems like it’s related to the issue you’re experiencing, but you're not sure, don’t be shy about posting on that Github issue to ask.  We can redirect it to a different issue if necessary.
+
+Pro-tip: if your Github issue includes a code snippet, add syntax highlighting by specifying the language at the top of the block:
```

Comment:
Thoughts on making this stand out? Perhaps add bold to `**Pro-tip**`, or consider formatting this in a callout. Happy to do that in a followup PR if you'd just prefer to get this out there.